tests/deepspeed/test_model_zoo.py

```diff
@@ -29,14 +28,11 @@
 )
 from transformers.trainer_utils import set_seed
 
+from ..trainer.test_trainer import TrainerIntegrationCommon  # noqa
 
-tests_dir = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
-root_dir = os.path.dirname(tests_dir)
-with ExtendSysPath(tests_dir):
-    from test_trainer import TrainerIntegrationCommon  # noqa
 
-    if is_torch_available():
-        from test_trainer import RegressionModelConfig, RegressionPreTrainedModel, get_regression_trainer  # noqa
+if is_torch_available():
+    from ..trainer.test_trainer import RegressionModelConfig, RegressionPreTrainedModel, get_regression_trainer  # noqa
```

Comment:
@stas00 pinging you on this explicitely, this is not needed anymore as it became a submodule. Please let me know if this doesn't work for you.

Reply:
thank you for the heads up, Lysandre

this part is fine, the one I flagged below is not.